<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IOFrame</class>
 <widget class="QFrame" name="IOFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabIOFrame">
     <widget class="QWidget" name="tabDigitalIO">
      <attribute name="title">
       <string>Digital IO</string>
      </attribute>
      <layout class="QGridLayout" name="glDigitalIO">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QScrollArea" name="saDigitalIO">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="saDigitalIOContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>820</width>
            <height>677</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="glDigitalIOContents">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <property name="leftMargin">
            <number>40</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>40</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="horizontalSpacing">
            <number>40</number>
           </property>
           <property name="verticalSpacing">
            <number>0</number>
           </property>
           <item row="1" column="0" colspan="2">
            <spacer name="vsDigitalIO">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>661</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="lDigitalInput">
             <property name="text">
              <string>Digital Input</string>
             </property>
             <property name="type" stdset="0">
              <string>subheader</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="lDigitalOutput">
             <property name="text">
              <string>Digital Output</string>
             </property>
             <property name="type" stdset="0">
              <string>subheader</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAnalogIO">
      <attribute name="title">
       <string>Analog IO</string>
      </attribute>
      <layout class="QGridLayout" name="glAnalogIO">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QScrollArea" name="saAnalogIO">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="saAnalogIOContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>820</width>
            <height>677</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="glAnalogIOContents">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <property name="leftMargin">
            <number>40</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>40</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="horizontalSpacing">
            <number>40</number>
           </property>
           <property name="verticalSpacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="lAnalogInput">
             <property name="text">
              <string>Analog Input</string>
             </property>
             <property name="type" stdset="0">
              <string>subheader</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <spacer name="vsAnalogInput">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>661</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
